### PR TITLE
Content: Increased warrior ceremonies

### PR DESCRIPTION
**Pull request by Tiritirimatangi**
## About The Pull Request

About 10 new elder and 30 new warrior ceremonies added

## Why This Is Good For ClanGen

More variety

## Proof of Testing

image

## Changelog/Credits

New warrior ceremonies

### CODE REVIEW

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 1]
```diff
@@ -2789,6 +2789,375 @@
...
+      "faithful",
+      "loving",
+      "ambitious",
+      "lonesome",
+      "strange",
+      "thoughtful",
+      "wise",
+      "sincere"
+    ],
+    "As (old_name) steps forward to take up the mantle of m_c, {PRONOUN/m_c/subject} {VERB/m_c/feel/feels} and wind ruffle through {PRONOUN/m_c/poss} fur, like a touch of approval. {PRONOUN/m_c/subject/CAP} {VERB/m_c/wonder/wonders} if dead_par1 would approve of the r_h m_c is honored for."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: "feels and wind ruffle" should most likely be "feels the wind ruffle"

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 2]
```diff
@@ -2789,6 +2789,375 @@
...
+      "alive2_parents",
+      "general_leader",
+      "general_backstory",
+      "childish",
+      "lonesome",
+      "insecure",
+      "nervous",
+      "grumpy"
+    ],
+    "(old_name) meets p1 and p2's eyes as {PRONOUN/m_c/subject} {VERB/m_c/swell/swells} with pride, taking {PRONOUN/m_c/poss} hard earned place in the spotlight in front of all of c_n. Given the name m_c in honor of {PRONOUN/m_c/poss} r_h, {PRONOUN/m_c/subject} {VERB/m_c/bask/basks} in the celebration of {PRONOUN/m_c/poss} warrior name, {PRONOUN/m_c/poss} family by {PRONOUN/m_c/poss} side."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: should be hard-earned

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 3]
```diff
@@ -2789,6 +2789,375 @@
...
+      "daring",
+      "righteous",
+      "adventurous",
+      "confident",
+      "shameless",
+      "arrogant",
+      "competitive",
+      "flamboyant"
+    ],
+    "Named m_c for {PRONOUN/m_c/poss} r_h, the new warrior squares {PRONOUN/m_c/poss} shoulders before c_n, heart full of the hard won joy of {PRONOUN/m_c/poss} Clanmates yowling {PRONOUN/m_c/poss} name, of p1's happy slitted eyes of pride and p2's voice leading the call. All {PRONOUN/m_c/poss} efforts to get here have been worth it. This is just the beginning."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: should be hard-won

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 4]
```diff
@@ -2789,6 +2789,375 @@
...
+      "alive2_parents",
+      "yes_leader",
+      "general_backstory",
+      "insecure",
+      "nervous",
+      "calm",
+      "careful",
+      "gloomy"
+    ],
+    "As l_n calls c_n together, (old_name) desperately suppresses {PRONOUN/m_c/poss} shivers. What if {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} not ready? What if this is a mistake? But hearing p1 yowl frantically to p2 to hurry up helps (old_name) find reserves of calm. {PRONOUN/m_c/subject/CAP} {VERB/m_c/belong/belongs} here, and stepping up to be honoured for {PRONOUN/m_c/poss} r_h with the name m_c fills {PRONOUN/m_c/object} with confidence."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: honoured should be americanized to honored

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 5]
```diff
@@ -2789,6 +2789,375 @@
...
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "strict",
+      "responsible"
+    ],
+    "With c_n gathered for celebration, l_n highlights (old_name)'s r_h, {PRONOUN/m_c/poss} dedication to truth and the warrior code. It is for this that l_n names {PRONOUN/m_c/object} m_c, welcoming {PRONOUN/m_c/object} as a full warrior of c_n."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: warrior code should be Warrior Code. also a missing oxford comma in the first sentence (r_h, their dedication , **and** the ...

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 6]
```diff
@@ -2789,6 +2789,375 @@
...
+      "prepared",
+      "alive_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "childish",
+      "oblivious",
+      "insecure"
+    ],
+    "l_n calls c_n to attention. They are gathered here to celebrate the hard work, persistence, and r_h of c_n's newest warrior. (old_name)'s jaw drops as {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} summoned forward, and {PRONOUN/m_c/subject} {VERB/m_c/sit/sits} , disbelieving but thrilled, as l_n grants {PRONOUN/m_c/object} the name m_c and cheers break out."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: {PRONOUN/m_c/subject} {VERB/m_c/sit/sits} ,

there shouldnt be a space between the sit verb tag and the comma

**Review by SunlitFable — APPROVED** [review]
all looks good to me! (with selkirk's changes that my brain glazed over)

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 7]
```diff
@@ -2789,6 +2789,375 @@
...
+      "alive_mentor",
+      "alive2_parents",
+      "yes_leader",
+      "general_backstory",
+      "grumpy",
+      "gloomy",
+      "cold",
+      "strange"
+    ],
+    "Even at {PRONOUN/m_c/poss} own naming ceremony, m_c doesn't love being the center of attention, the tip of {PRONOUN/m_c/poss} tail twitching with unease. But {PRONOUN/m_c/subject} still {VERB/m_c/accept/accepts} (previous_mentor)'s congratulations and {PRONOUN/m_c/poss} parents proud purrs, still smiles slightly when l_n praises {PRONOUN/m_c/poss} r_h."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: parents should be parents'

Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: the second sentence reads as
But she still accepts mentor's congratulations and her parents' proud purrs, **still smiles** slightly when l_n praises her r_h
the bolded part should be written, probably to "smiling slightly"

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 8]
```diff
@@ -2937,5 +3306,86 @@
...
+  "elder_6": [
+    [
+      "elder",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "all_traits"
+    ],
+    "m_c holds {PRONOUN/m_c/poss} head up high as l_n calls upon StarClan to witness m_c's service to c_n. This is a retirement hard earned."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: should be hard-earned

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 9]
```diff
@@ -2937,5 +3306,86 @@
...
+  "elder_7": [
+    [
+      "elder",
+      "general_mentor",
+      "general_parents",
+      "general_leader",
+      "general_backstory",
+      "all_traits"
+    ],
+    "As m_c entering into the sunset of {PRONOUN/m_c/poss} life, {PRONOUN/m_c/subject} {VERB/m_c/join/joins} the ranks of the elders without regret, looking forward to the rest {PRONOUN/m_c/subject} {VERB/m_c/have/has} earned."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: "as m_c entering into the sunset..." should be written as "as m_c enters into the sunset..." or something similar

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 10]
```diff
@@ -2937,5 +3306,86 @@
...
+      "general_mentor",
+      "general_parents",
+      "general_leader",
+      "general_backstory",
+      "responsible",
+      "ambitious",
+      "strict",
+      "insecure"
+    ],
+    "m_c stops {PRONOUN/m_c/poss} tail from twitching as {PRONOUN/m_c/poss} retirement is announced. {PRONOUN/m_c/subject/CAP} {VERB/m_c/know/knows} c_n will be fine without {PRONOUN/m_c/object} on the front lines, {PRONOUN/m_c/subject} know this is the right time to relax {PRONOUN/m_c/poss} claws and let {PRONOUN/m_c/poss} responsibilities fall from {PRONOUN/m_c/poss} paws, but... it's hard, letting go."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: in the second line, know should be verb tagged

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 11]
```diff
@@ -2937,5 +3306,86 @@
...
+      "general_mentor",
+      "general_parents",
+      "general_leader",
+      "general_backstory",
+      "gloomy",
+      "nervous",
+      "lonesome",
+      "insecure"
+    ],
+    "m_c takes a deep breath, and tries to see the celebration of {PRONOUN/m_c/poss} retirement as a compliment, not an exile to the elder's den. {PRONOUN/m_c/subject/CAP} {VERB/m_c/know/knows} it isn't meant that way."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: should be elders' den

File: resources/dicts/events/ceremonies/ceremony-master.json [hunk 12]
```diff
@@ -2937,5 +3306,86 @@
...
+      "general_leader",
+      "general_backstory",
+      "loyal",
+      "righteous",
+      "fierce",
+      "bloodthirsty",
+      "vengeful",
+      "sincere"
+    ],
+    "m_c sits with a puffed out pelt of pride, hearing c_n celebrate {PRONOUN/m_c/poss} achivements. Retiring means trusting younger, fresher cats to take up the fight to keep c_n safe, but {PRONOUN/m_c/subject} {VERB/m_c/think/thinks} {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} done the best {PRONOUN/m_c/subject} could've to pass on {PRONOUN/m_c/poss} lessons."
```
Comment (selkirks) [inline resources/dicts/events/ceremonies/ceremony-master.json]: achievements is spelt wrong

File: resources/dicts/patrols/plains/hunting/any.json [hunk 13]
```diff
@@ -440,23 +512,151 @@
...
+            },
+            {
+                "text": "This warren was a bad call, and its rabbits twitch out of sight with the vigilence of the recently-hunted. Though p_l picks up a mouse on the way home, {PRONOUN/p_l/subject}{VERB/p_l/'re/'s} still disappointed.",
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_hunt_sneakprey",
+                "prey": ["very_small"]
+            },
+            {
+                "text": "Bolds, s_c charges towards a feeding bunny. And sure, s_c gets <i>{PRONOUN/s_c/poss}</i> rabbit, but what about the rest of the patrol? It does not leave the rest of them feeling particularly impressed.",
```
Comment (selkirks) [inline resources/dicts/patrols/plains/hunting/any.json]: Bolds should be Boldly

**Review by selkirks — DISMISSED** [review]

File: resources/dicts/patrols/plains/hunting/any.json (hunk 13 above)
Comment (Tiritirimatangi) [reply resources/dicts/patrols/plains/hunting/any.json]: thank you so much for typo hunting! They should all be fixed now <3

**Pull request merged**